cli/command/stack/kubernetes/loader.go

```diff
@@ -1,18 +1,34 @@
 package kubernetes
 
 import (
+	"github.com/docker/cli/cli/compose/loader"
+	"github.com/docker/cli/cli/compose/schema"
 	composetypes "github.com/docker/cli/cli/compose/types"
 	apiv1beta1 "github.com/docker/cli/kubernetes/compose/v1beta1"
+	"github.com/pkg/errors"
 	yaml "gopkg.in/yaml.v2"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const v1beta1MaxComposeVersion = "3.5"
```

Comment:
Can we move this constant to the apiv1beta1 package? It should be the package responsibility to know which compose version it can handle.

Reply:
And maybe we should add one also for v1beta2, and generalize the process?

Reply:
Good point 😅